### ISSUE TITLE
add-graphql-datasource not supporting the ap-southeast-2 region

### ISSUE THREAD

**Issue by rajcrystaldelta** [aws-amplify/amplify-cli#4739]
**Describe the bug**
when run `amplify api add-graphql-datasource`
currently it shows only 5 regions
```
  us-east-1
  us-east-2
  us-west-2
  ap-northeast-1
  eu-west-1
```

**Amplify CLI Version**
4.22.0

**To Reproduce**
1. Initialize a new Amplify project - `amplify init` in ap-southeast-2 region
2. Add api - `amplify add api`
3. Create Aurora serverless RDS as per [Documentation](https://docs.amplify.aws/cli/graphql-transformer/relational) in ap-southeast-2 region and enabled Data API option
4. add aurora datasource - `amplify api add-graphql-datasource`
Asia Pacific (Sydney)ap-southeast-2 region is not in the option
```
Using datasource: Aurora Serverless, provided by: awscloudformation
? Provide the region in which your cluster is located:
❯ us-east-1
  us-east-2
  us-west-2
  ap-northeast-1
  eu-west-1
```

**Expected behavior**
Asia Pacific (Sydney)ap-southeast-2 region also supports the Aurora serverless with Data API [reference](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html) then it should be listed in the options. 
Eg: 
```
Using datasource: Aurora Serverless, provided by: awscloudformation
? Provide the region in which your cluster is located:
❯ us-east-1
us-east-2
us-west-1
us-west-2
ap-south-1
ap-southeast-1
ap-southeast-2
ap-northeast-1
eu-central-1
eu-west-1
eu-west-2
eu-west-3
```

**Screenshots**
<img width="752" alt="terminal-screenshot" src="https://user-images.githubusercontent.com/32514653/86465401-28a21b80-bd4f-11ea-8256-5aecabb761f6.png">

**Desktop (please complete the following information):**
 - OS: Mac
 - Node Version: v12.16.0

**Comment by berenddeboer** [comment]
Only those regions support RDS HTTP endpoint.

**Comment by rajcrystaldelta** [comment]
hello @berenddeboer,
As per document [Data API for Aurora document](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html), RDS HTTP endpoint in available in ap-southeast-2 region.

Also, I tested in AWS console in ap-southeast-2 region, I can enable Data API ( HTTP endpoint ).
Refer the following console screenshot,
<img width="1664" alt="Screenshot 2020-07-05 at 4 30 39 PM" src="https://user-images.githubusercontent.com/32514653/86531287-8c5a4f00-bedd-11ea-94fe-dcb4fbd13790.png">

**Comment by ngchwanlii** [comment]
Please also add support for ap-southeast-1 region. Thanks!

**Comment by minu93** [comment]
When this issue will be fixed?
We need a fix urgant. 
![image](https://user-images.githubusercontent.com/48417250/87805699-10092980-c856-11ea-92ed-6e90bd3deaf6.png)
We need eu-centrel-1

Thanks a lot !

**Comment by micduffy** [comment]
Please note, very interested in this item being resolved also

**Comment by rajcrystaldelta** [comment]
@renebrandel - As we need this urgent, can you review the PR and take action, please.

**Comment by renebrandel** [comment]
@rajcrystaldelta sorry missed this issue before! Let me verify that there aren't any downstream implications. Will try to get back with next steps by Friday!

**Comment by tomaseti** [comment]
@renebrandel Yep with Brexit, and GDPR regulations we desperately need this functionality.

**Comment by renebrandel** [comment]
@tomaseti - we looked at the PR and the rebasing is gonna get a bit messy. (sorry about our slow response on that!) We're looking to create a PR ourselves to address this. I'll keep this thread up to date.

**Comment by yuth** [comment]
The API deployment would fail as the necessary changes needed in the AppSync service is missing. The AppSync service is actively working to support these additional regions and we will pick this up once the service starts supporting them

**Comment by tomaseti** [comment]
@yuth Do you have any eta, when this will done?  We're waiting to use this functionality in the London region.

**Comment by yuth** [comment]
I don't have an ETA, but the AppSync service team is actively working on this

**Comment by yuth** [comment]
The fix was released in @aws-amplify/cli@4.29.4

**Comment by tomaseti** [comment]
Great news thank you
